### ISSUE TITLE
Laravel 11 breaks php artisan orion:spec

### ISSUE THREAD

**Issue by dragosstoenica** [tailflow/laravel-orion#259]
```
php artisan orion:specs
```

 outputs 
  
```
BadMethodCallException

  Method Illuminate\Database\MySqlConnection::getDoctrineSchemaManager does not exist.

  at vendor/laravel/framework/src/Illuminate/Macroable/Traits/Macroable.php:112
    108▕      */
    109▕     public function __call($method, $parameters)
    110▕     {
    111▕         if (! static::hasMacro($method)) {
  ➜ 112▕             throw new BadMethodCallException(sprintf(
    113▕                 'Method %s::%s does not exist.', static::class, $method
    114▕             ));
    115▕         }
    116▕

  i   Bad Method Call: Did you mean Illuminate\Database\MySqlConnection::getSchemaGrammar() ?

      +20 vendor frames

  21  artisan:13
      Illuminate\Foundation\Application::handleCommand(Object(Symfony\Component\Console\Input\ArgvInput))
```

Laravel and Laravel-Orion versions
```
"laravel/framework": "^11.0",
"tailflow/laravel-orion": "^2.2"
```

**Comment by alexzarbn** [comment]
Hi @dragosstoenica,

Thank you for reporting this! I will look into it.

**Comment by alexzarbn** [comment]
The issue is related to [Doctrine DBAL Removal](https://laravel.com/docs/11.x/upgrade#doctrine-dbal-removal) in Laravel 11.

An updated implementation with support for native schema methods is available in [v2.22.4](https://github.com/tailflow/laravel-orion/releases/tag/2.22.4) release.

@dragosstoenica Please let me know, if you encounter further issues with specs generation on Laravel 11.

**Comment by alberto-bottarini** [comment]
To confirm this is fixed
